backend/src/apiserver/config/sample_config.json

```diff
@@ -16,7 +16,7 @@
   },
   {
     "name": "[Tutorial] DSL - Control structures",
-    "description": "[source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures) Shows how to use conditional execution and exit handlers.",
+    "description": "[source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures) Shows how to use conditional execution and exit handlers. This pipeline will randomly fail to demonstrate that exit handler gets executed even in case of failure.",
```

Comment:
Nit: 

that exit handler -> that the exit handler

Reply:
FIxed